dashboard/test/ui/features/hamburger.feature

```diff
@@ -0,0 +1,150 @@
+Feature: Hamburger dropdown
+
+  Scenario: Signed out user in English should not see hamburger on desktop
+    Given I am on "http://code.org/"
+    And I set the language cookie
+    Then I wait until element "#hamburger-icon" is not visible
+
+  Scenario: Student viewing hamburger dropdown in English on desktop on Code.org
+    Given I am on "http://code.org/"
+    And I set the language cookie
+    And I create a student named "Sally Student"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see ".hide-responsive-menu"
+    And I see "#report-bug"
+    And I see "#support"
+    And I see ".divider#before-pegasus"
+    And I see "#learn"
+    And I see "#educate-more"
+    And I see "#stats"
+    And I see "#help_us"
+    And I see "#about-more"
+
+  Scenario: Teacher viewing hamburger dropdown (with expanded options) in English on desktop on Code.org
+    Given I am on "http://code.org/"
+    And I set the language cookie
+    And I create a teacher named "Tessa Teacher"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see ".hide-responsive-menu"
+    And I see "#report-bug"
+    And I see "#support"
+    And I see "#teacher-community"
+    And I see ".divider#before-pegasus"
+    And I see "#learn"
+    And I see "#educate-more"
+    And I see "#stats"
+    And I see "#help_us"
+    And I see "#about-more"
+    And I click selector "#educate-more"
+    And I see "#educate-link"
```

Comment:
Note that it might be in the process of expanding at this point.  Probably okay in this case but worth keeping an eye out.